### PR TITLE
[#291] Added generic scan workflow.

### LINKED ISSUE #291: Code scanning items are not attributed to the image tag they exist on
**Describe the bug**
When Trivy scheduled scans are executed they run on both the 5.x and 6.x branches however on the Code Scanning dashboard the associated CVEs are attributed to the branch that ran the workflow. For scheduled events that is whatever the default branch is set to.

**To Reproduce**
Execute the workflow [vulnerability-scan-schedule-5.x.ym](https://github.com/dpc-sdp/bay/blob/feature/specify-branch-on-scan/.github/workflows/vulnerability-scan-schedule-5x.yml)

In an attempt to achieve the desired behaviour I've created a [reusable workflow](https://github.com/dpc-sdp/bay/blob/feature/specify-branch-on-scan-5.x/.github/workflows/vulnerability-scan-schedule.yml#L19) in branch 'A' that is called [from another branch](https://github.com/dpc-sdp/bay/blob/feature/specify-branch-on-scan/.github/workflows/vulnerability-scan-schedule-5x.yml), branch 'B'

**Expected behavior**
The [code scanning advisories listing](https://github.com/dpc-sdp/bay/security/code-scanning) attributes the advisory to the ref/tag of the package or ref/tag of the actions/checkout step

Using the above example of branch 'A' and 'B' I would expect the advisories to be attributed to branch 'B'

A further example
i.e. from the workflow in the [feature/specify-branch-on-scan-5.x branch](https://github.com/dpc-sdp/bay/blob/5c80ad79f57dc39faba4c9e1bdacdcce4dd9341d/.github/workflows/vulnerability-scan-schedule.yml#L19)

```
      - uses: actions/checkout@v4
        with:
          ref: 5.x
```

I would expect/desire that the advisories in code scanning are attributed to 5.x

**Actual behaviour**
The [code scanning advisories listing](https://github.com/dpc-sdp/bay/security/code-scanning) attributes the advisory to the ref/tag of the branch that executed the workflow, in the below example 6.x

### PR DESCRIPTION
**Pull request by GROwen**
### Fixes
#291 

### Related PRs
dpc-sdp/sdp-ansible-playbooks#

### CODE REVIEW

File: images/php/Dockerfile.cli [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM ghcr.io/skpr/mtk:v2.0.2 AS mtk
 FROM uselagoon/php-${PHP_VERSION}-cli-drupal:latest
 
 ARG GOJQ_VERSION=0.12.16
-ARG DOCKERIZE_VERSION=v0.8.0
+ARG DOCKERIZE_VERSION=v0.6.1
```
Comment (nicksantamaria) [inline images/php/Dockerfile.cli:7]: Is this fixed elsewhere?

**Review by nicksantamaria — APPROVED** [review]

File: images/php/Dockerfile.cli (hunk 1 above)
Comment (GROwen) [reply images/php/Dockerfile.cli:7]: yep, #295

**Pull request merged**